store/src/java/com/zimbra/cs/ldap/unboundid/UBIDAttributes.java

```diff
@@ -105,7 +104,11 @@ protected String getAttrString(String transferAttrName, boolean containsBinaryDa
     protected String[] getMultiAttrString(String transferAttrName, boolean containsBinaryData) 
     throws LdapException {
         Attribute attr = entry.getAttribute(transferAttrName);
-        
+        // AD sends 'userCertificate;binary' attribute as 'userCertificate' without appending ';binary' to it
+        if (transferAttrName.contains(ContactConstants.A_userCertificate) && attr == null) {
+            attr = entry.getAttribute(transferAttrName.replace(";binary", ""));
```

Comment:
no need of replace, can use ContactConstants.A_userCertificate directly as attribute name.